src/pal/src/CMakeLists.txt

```diff
@@ -282,7 +282,7 @@ if(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
   set(LIBUNWIND_OBJECTS $<TARGET_OBJECTS:libunwind>)
 endif(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
 
-add_library(coreclrpal
+add_library_clr(coreclrpal
```

Comment:
This will add the version file to the PAL. It seems that this way, all the shared libraries and applications we build would get it twice. I'm not sure what would be the result of this. Can you please scan the binaries of .so files, corerun etc using the `strings xxxx | grep "@(#)"` to check that the version info is not included twice in them?

Reply:
@janvorli Thank you for pointing this out! You are right, the version info indeed included twice

Reply:
I wonder what would be the appropriate way to disable libcoreclrpal from building as crosscomponent

Reply:
@janvorli It seems that to exclude coreclrpal from publishing as crosscomponent replacing `install (TARGETS coreclrpal` with `_install (TARGETS coreclrpal` is sufficient. Are you fine with this change?

Reply:
That's fine, I guess that the fact that it would still get built is fine. Or, maybe it would not get built because there would be no install for it. I am not sure.

Reply:
libcoreclrpal.a is still got built as a dependency of crossgen (an clrjit) - it just don't get published to `bin/Product/Linux.arm.{BuildType}/x86/lib` dir anymore.

Reply:
Ah, ok, thanks for the details.